From 1e9a341fa58f68335efef0e94111c4eee619cdab Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Genezys=20Marques=20Gon=C3=A7alves?= <genezys@gmail.com>
Date: Sun, 14 Dec 2014 04:21:40 -0200
Subject: [PATCH] Initial commit

---
 README.md | 4 ++++
 1 file changed, 4 insertions(+)
 create mode 100644 README.md

diff --git a/README.md b/README.md
new file mode 100644
index 0000000..fe5d3e5
--- /dev/null
+++ b/README.md
@@ -0,0 +1,4 @@
+tcc
+===
+
+Trabalho de Conclusão de Curso
-- 
1.9.1

